### DISCUSSION TITLE
Feature ideas

### DISCUSSION THREAD

**Discussion by prathamesh-sonpatki** [prathamesh-sonpatki/o11y-wiki#9] [Ideas]
- Individual pages for a term
- Ability to add alternatives
- Interesting talks
- Related blogs
- Related terms

**Comment by santiagolator** [comment]
I love the project 😃 Being Spanish-speaking, it would be ideal if we could have a Spanish version. I could help with translations!

**Reply by prathamesh-sonpatki** [reply]
@santiagolator Sure. How do you want to proceed with this?

**Reply by prathamesh-sonpatki** [reply]
They recommend something like this https://docsify.js.org/#/cover?id=multiple-covers Let me try this in next couple of days and come back :)
